tests/migrations/test_operations.py

```diff
@@ -2068,6 +2076,52 @@ def test_remove_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {"model_name": "Pony", "name": "pink"})
 
+    def test_remove_field_blocked(self):
+        """
+        Tests migration is blocked by dependent objects.
+        """
+        project_state = self.set_up_test_model("test_rmflb")
+        new_state = project_state.clone()
+
+        with connection.cursor() as cursor:
```

Comment:
I think it might be nice to define a cleanup to drop the view, a bit like:
```python
        view_name = "test_rmflb_pony_view"

        def drop_view():
            with connection.cursor() as c:
                c.execute(f"DROP VIEW {view_name}")

        with connection.cursor() as cursor:
            cursor.execute(
                f"CREATE VIEW {view_name} AS SELECT pink FROM test_rmflb_pony"
            )
        self.addCleanup(drop_view)
```